### PR TITLE
Add password to root user and set no password auth for ssh config

### PR DESCRIPTION
**Pull request by rogan7n**
MZBench workers cannot be ssh'd into due to missing password on root user. 

```Stage provisioning failed: {cmd_failed, "ssh -A -o LogLevel=ERROR -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no root@100.65.195.111 \"source /etc/profile; mkdir -p /tmp/mz/bench-0-1562785108\"", 255, "Permission denied, please try again.\r\nPermission denied, please try again.\r\nroot@100.65.195.111: Permission denied (publickey,password,keyboard-interactive).\r\n"}```

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -70,6 +70,9 @@ RUN apk add --no-cache libstdc++ git curl openssh openssh-server bash rsync net-
 COPY --from=build $MZBENCH_API_DIR $MZBENCH_API_DIR
 COPY --from=build ${HOME_DIR}/.local ${HOME_DIR}/.local
 
+# SSH requires root to have a password. Otherwise it will prompt for an interactive password
+RUN echo "root:Docker!" | chpasswd && "PasswordAuthentication no" > /etc/ssh/sshd_config
```
Comment (SysOfM) [inline Dockerfile]: In order not to create an additional layer could you move these command to one-liner above, something like this?
  
```
&& chmod 0600 ${HOME_DIR}/.ssh/authorized_keys \ 
&& echo "root:Docker!" | chpasswd && "PasswordAuthentication no" > /etc/ssh/sshd_config \
&& pip install -r /tmp/requirements.txt \
```

**Comment by gliush** [comment]
Don't you want to continue your previous PR that contains the same change as well as some other important changes? :)
https://github.com/mzbench/mzbench/pull/6

**Comment by rogan7n** [comment]
> Don't you want to continue your previous PR that contains the same change as well as some other important changes? :)
> #6

Yup that PR does have important changes, but I haven't had time to address the comments. I will try to get that in by EOW. In the meantime, I just want to unblock the other devs from performing tests.

**Comment by SysOfM** [comment]
Yet one thing, you forgot to add `echo` to the command.
 This shuould be like:
`echo "PasswordAuthentication no" > /etc/ssh/sshd_config`

**Comment by rogan7n** [comment]
> Yet one thing, you forgot to add `echo` to the command.
> This shuould be like:
> `echo "PasswordAuthentication no" > /etc/ssh/sshd_config`

Good catch @SysOfM . Fixed.

**Comment by pulltab** [comment]
@rogan7n @SysOfM @gliush Bump.

This should be merged.  I co-discovered this issue / fix this week.

**Review by gliush — APPROVED** [review]
Yep, LGTM, merging.

**Pull request merged**